modules/standard/IO.chpl

```diff
@@ -2653,7 +2478,7 @@ record defaultSerializer {
     parentheses. For example, the tuple literal ``(1, 2, 3)`` would be
     serialized as:
 
-    .. code-block::
+    .. code-block:: text
```

Comment:
Let's go ahead and have `example_serialized_class.chpl` use the serializer in order to print out the class. Then you can change this line to `.. literalinclude` the `.good` file for that test.

Here is a line I used when converting over the JSON tests:

```chapel
/* STOP_EXAMPLE_0 */
var c = new Child(x=5, y=2);
stdout.withSerializer(jsonSerializer).writeln(c);
```

Perhaps we can replace `jsonSerializer` in that last chain of calls with `defaultSerializer` instead.